test/unit_interior.jl

```diff
@@ -0,0 +1,6 @@
+P = BallInf(zeros(2), 0.15)
+d = [0.15,0.15]
+@test !is_interior_point(d, P)
+@test !is_interior_point(d, P; p=2.0)
+d = [0.15,0.15] .- LazySets._TOL_F64.rtol
+@test is_interior_point(d, P)
```

Comment:
Can you add a positive and a negative test for modifying `p` resp. `ε`?
```julia
@test is_interior_point(d, P; p=2)
d = [0.15, 0.15]
@test !is_interior_point(d, P; ε=1.0)
@test is_interior_point(d, P; ε=0.0)
```

Reply:
`Ballp` does only work if `p` has the same type as the `center` and the `radius`. 
So your first test with `p=2` does not work.
Is there a specific reason for that?

Reply:
>Ballp does only work if p has the same type as the center and the radius.

Okay, makes sense.

> Is there a specific reason for that?

No, I just forgot that it needs to be a float.